Services/Seed.cs

```diff
@@ -162,7 +162,14 @@ public class Seed : ISeed
             {
                 try
                 {
-                    await this.telemetryClient.UpdateRuleAsync(rule, "*");
+                    if (null == await this.telemetryClient.GetRuleAsync(rule.Id))
+                    {
+                        await this.telemetryClient.CreateRuleAsync(rule);
```

Comment:
Will this generate a new random ID for the rule?

Reply:
Yes it will generate the random id for the rule.